lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -37,6 +35,7 @@ def to_s(xml_declaration = true)
         return ('<?xml version="1.0" encoding="UTF-8"?>' + xml) if xml_declaration
         xml
       end
+      alias to_xml to_s
```

Comment:
There's no controversy, just the right way to use each of the methods, which is as described. Aliasing `to_s` to `to_xml` here is perfectly clear.

I think it would look un-Ruby to just do `def to_xml ; to_s ; end`. It wouldn't be order defined,  `to_s` would be looked up dynamically at runtime.